build-tools/mono-runtimes/mono-runtimes.projitems

```diff
@@ -1,7 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+      Each _MonoRuntime is placed in its own ItemGroup to keep it together with the associated PropertyGroup that
```

Comment:
One thing to keep in mind -- but need not result in any change to this PR -- is that MSBuild will evaluate `<PropertyGroup/>` before `<ItemGroup/>`s. This means that a toplevel `<PropertyGroup/>` value *cannot* reference an `<ItemGroup/>` value, because the item groups won't have been evaluated yet.

My conceptual but ignorable problem with intermixing property groups and item groups is that it's easy to not know or forget the above behavior, and so one might inadvertently have the property group rely on item group values, which won't work.